onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -103,85 +103,6 @@ TEST(TensorrtExecutionProviderTest, FunctionTest) {
   VerifyOutputs(fetches, expected_dims_mul_m, expected_values_mul_m);
 }
 
-TEST(TensorrtExecutionProviderTest, DynamicShapeTest) {
```

Comment:
why is this removed?

Reply:
It's not needed since we added dynamic shape model in the mode tests for both Linux and Windows.